tests/storage/export.go

```diff
@@ -1894,13 +1893,17 @@ var _ = SIGDescribe("Export", func() {
 		Expect(err).ToNot(HaveOccurred())
 		diskDV.Name = fmt.Sprintf("%s-clone", diskDV.Name)
 		diskDV.Spec.Storage.StorageClassName = virtpointer.P(sc)
-		Expect(diskDV.Spec.Storage.Resources.Requests[k8sv1.ResourceStorage]).To(BeEquivalentTo(resource.MustParse(cd.CirrosVolumeSize)))
-		blankDv = &cdiv1.DataVolume{}
-		err = yaml.Unmarshal([]byte(split[3]), blankDv)
+		diskPVC, err := virtClient.CoreV1().PersistentVolumeClaims(testsuite.GetTestNamespace(dataVolume)).Get(context.Background(), dataVolume.Name, metav1.GetOptions{})
 		Expect(err).ToNot(HaveOccurred())
-		blankDv.Name = fmt.Sprintf("%s-clone", blankDv.Name)
-		blankDv.Spec.Storage.StorageClassName = virtpointer.P(sc)
-		Expect(blankDv.Spec.Storage.Resources.Requests[k8sv1.ResourceStorage]).To(BeEquivalentTo(resource.MustParse(cd.BlankVolumeSize)))
+		Expect(diskDV.Spec.Storage.Resources.Requests[k8sv1.ResourceStorage]).To(BeEquivalentTo(diskPVC.Spec.Resources.Requests[k8sv1.ResourceStorage]))
+		blankDiskDV := &cdiv1.DataVolume{}
+		err = yaml.Unmarshal([]byte(split[3]), blankDiskDV)
+		Expect(err).ToNot(HaveOccurred())
+		blankDiskDV.Name = fmt.Sprintf("%s-clone", blankDv.Name)
+		blankDiskDV.Spec.Storage.StorageClassName = virtpointer.P(sc)
```

Comment:
Pardon my ignorance, but can't you take the address of `sc` without `virtpointer.P`?

Reply:
Yes!